packages/ui/src/components/ExpandableCard/__stories__/index.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import type { Meta } from '@storybook/react'
+import { ExpandableCard } from '..'
+
+export default {
+  component: ExpandableCard,
+  title: 'Components/Layout/ExpandableCard',
+  subcomponents: {
+    'ExpandableCard.Title': ExpandableCard.Title,
+  },
+} as Meta
```

Comment:
`satisfies Meta` ?

Reply:
with the `subComponents` the typing do not work